inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -703,3 +704,37 @@ def increase_restart_count_or_error(obj, eng):
         save_workflow(obj, eng)
     else:
         raise WorkflowsError('Workflow restarted too many times')
+
+
+def flatten_list(input_list):
+    if isinstance(input_list, (list, tuple)):
+        return [
+            element for innerList in input_list for element in flatten_list(innerList)
+        ]
+    return [input_list]
+
+
+@with_debug_logging
+def affiliations_for_hidden_collections(obj):
+    affiliations_mapping = current_app.config.get("AFFILIATIONS_TO_HIDDEN_COLLECTIONS_MAPPING", {})
+    affiliations = flatten_list(get_value(obj.data, 'authors.raw_affiliations.value', []))
+
+    query = "|".join(affiliations_mapping.keys())
```

Comment:
I said word boundary for a specific reason, we don't want to match on things like `concern`
```suggestion
    query = "|".join(f"\b{key}\b" for key in affiliations_mapping.keys())
```

Reply:
Argh... Sorry, this comment appeared on the moment when I merged... I will create a fix.